content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -117,12 +122,75 @@ kind: KubeletConfiguration
 address: "192.168.0.8"
 ```
 
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+address: "192.168.0.8"
+```
+Additionally, you can overidde the slices/lists values of the kubelet configuration. For example, you can override the `clusterDNS` and `featureGates` fields as follows:
+
+Main kubelet configuration file contents:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.9"
+  - "192.168.0.8"
+featureGates:
+  MemoryQoS: true
+```
+
+Contents of a file in `--config-dir` directory:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+clusterDNS:
+  - "192.168.0.2"
+  - "192.168.0.3"
+  - "192.168.0.5"
+featureGates:
+  MemoryQoS: false
+  KubeletTracing: true
+```
+
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.2"
+  - "192.168.0.3"
+  - "192.168.0.5"
+featureGates:
```

Comment:
will the speciall `all/alpha=TRUE` work in drop-in config? Will it respect previously disabled alphas or will force enable them?

Reply:
>will the speciall all/alpha=TRUE work in drop-in config

It should work

>Will it respect previously disabled alphas or will force enable them?

The latter

Reply:
@sohankunkerkar can you include this in the website example?

Reply:
>Will it respect previously disabled alphas or will force enable them?

The above logic will be handled by the Kubelet. This functionality aims to replace the corresponding field value.